gensim/utils.py

```diff
@@ -829,15 +828,15 @@ def __init__(self, q, corpus, chunksize, maxsize, as_numpy):
 
     def run(self):
         if self.as_numpy:
-            import numpy # don't clutter the global namespace with a dependency on numpy
+            import np # don't clutter the global namespace with a dependency on numpy
```

Comment:
Looks like the global module namespace already has numpy, so this line is unnecessary.